pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -256,37 +256,6 @@ func (r *Retriever) store(u *unstructured.Unstructured, key string, value []byte
 	r.data[key] = value
 }
 
-// Retrieve loop and read data pointed by the references in plan instance. Also runs through
-// "bindable resources", gathering extra data. It can return error on retrieving and reading
-// resources.
-func (r *Retriever) Retrieve() (map[string][]byte, error) {
```

Comment:
@isutton Keeping this dead code would make me fix a lot of tests associated with this dead code. Curious, why do you want this code to be kept ? :)

Reply:
I reverted the changes and submitted a PR here: https://github.com/sbose78/service-binding-operator/pull/3

Since we want to have smaller PRs, I judged the changes from this dead code would add noise to your real changes.

Reply:
Thanks Igor, did you fix the tests too?